.env

```diff
@@ -0,0 +1,2 @@
+SESSION_SECRET="3fhfivo'+_#@V',>"
```

Comment:
Change this to something that looks less "random" so you won't confuse it for a real Session Secret.